skfuzzy/control/visualization.py

```diff
@@ -1,39 +1,63 @@
 """
-visualization.py : Contains classes to help with visualizing a control system
+visualization.py : Visualize fuzzy control systems.
 """
+from __future__ import print_function, division
+
 import numpy as np
 import networkx as nx
 import matplotlib.pyplot as plt
 
-from .. import defuzz, interp_membership
+from ..fuzzymath.fuzzy_ops import interp_membership
 
 
 class FuzzyVariableVisualizer(object):
+    """
+    Visualize a fuzzy variable and its membership functions.
+
+    Parameters
+    ----------
+    fuzzy_var : FuzzyVariable or Term
+        Fuzzy variable to be plotted.
+
+    Returns
+    -------
+    figure : matplotlib Figure
+        Figure object containing the visualization.
+    """
 
     def __init__(self, fuzzy_var):
         """
+        Initialize the fuzzy variable plot.
 
         Parameters
         ----------
-        fuzzy_var : FuzzyVariable to plot
+        fuzzy_var : FuzzyVariable or Term to plot
+        """
+        from .fuzzyvariable import FuzzyVariable, Term
 
-        Returns
-        -------
+        assert (isinstance(fuzzy_var, FuzzyVariable) or
+                isinstance(fuzzy_var, Term))
+
+        # self.term allows us to know if this is a Term quickly, later
+        self.term = None
+        if isinstance(fuzzy_var, Term):
+            self.term = fuzzy_var.label
+            self.fuzzy_var = fuzzy_var.parent_variable
+        else:
```

Comment:
I would use elif isinstance(fuzzy_var, FuzzyVariable), raise a `ValueError` in the else, and remove the assert.